### ISSUE TITLE
I/O labeling needs to be swapped?

### ISSUE THREAD

**Issue by venice1200** [SukkoPera/OpenAmigaMouseTrigger#4]
Many Thanks for this very useful gadget.
I use it successful with a button and your DriveSwitcher.

Since I use the PullUp R1 with my button, I had to use I2 as input and O2 as output.

I double checked the PCB and the LS74 Pinout an saw:
- Input I2 is connected to R1 and D1 (2) of the LS74
- Output O2 is connected to 1Q (5) of the LS74
- Input I1 is connected to R2 and D2 (12) of the LS74
- Output O2 is connected to 2Q (9) of the LS74

Is it possible that the labling of the I/O Pins need to be swapped?
I1<=>I2 and O1<=>O2

![IOS_Swapped_Question](https://github.com/SukkoPera/OpenAmigaMouseTrigger/assets/43762187/69e69355-903c-4964-9aff-574a0a771a47)

**Comment by SukkoPera** [comment]
You are right, the labels are incorrect. What is labeled I1/O1 is actually IN2/OUT2 on the schematics (and vice-versa).

I'm sorry for that but I am not going to make a new release just to fix that at the moment. Let's leave this open as a reminder.

I think it's safe to install the pull-ups in all cases anyway, there's really no need to wonder whether to do it or not.

**Comment by venice1200** [comment]
Many Thanks for your confirmation and your great projects.

All the best!

**Comment by kattan1969** [comment]
This for sure one of the best gadgets to use with my Amiga switching between KS 3.2 and the original 1.3 using the Open Kickstart switcher. However, when I used the 27C160 with four 512K images, the A19 signal doesn't hold and I get a crash! In other words, selecting the first two ROMs was not a problem but the 3rd and 4th works if I keep hold the relevant joystick button/direction and crashes as soon as I release it!! I will be grateful if you could help me with some feedback. Thanks

**Comment by SukkoPera** [comment]
That is quite odd, please double check the values of the resistors and cap. Maybe replace the IC. If you have a scope you can check what's going on on pin 3 when you power on the machine. If you need more help please open a dedicated issue.
